### PR TITLE
SearchInventory GUI

### PR DESCRIPTION
**Pull request by jblss888**
Not connected to the inventory yet, but it is functional

### CODE REVIEW

File: mySqlDatabaseRemote/src/mySqlDriver2/SearchInventoryGUI.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+               record = searchSid.getText();
+            }
+            //System.out.println(record + " " + fieldselection);
+        }
+     });
+      cp.add(submitButton);
+ 
+      // Allocate an anonymous instance of an anonymous inner class that
+      //  implements ActionListener as ActionEvent listener
+      submitButton.addActionListener(new ActionListener() {
```
Comment (Naomimmm) [inline mySqlDatabaseRemote/src/mySqlDriver2/SearchInventoryGUI.java:107]: Is this one different from the action listener above? Is this one needed too?

File: mySqlDatabaseRemote/src/mySqlDriver2/SearchInventoryGUI.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+               record = searchPid.getText();
+               
+           }
+           else if(fieldselection == 2 && searchTitle.getText() != null){
+               record = searchTitle.getText();
+           }
+           else if(fieldselection == 5 && searchSid.getText() != null){
+               record = searchSid.getText();
+            }
+            //System.out.println(record + " " + fieldselection);
```
Comment (Naomimmm) [inline mySqlDatabaseRemote/src/mySqlDriver2/SearchInventoryGUI.java:100]: Do we need a dispose() here?

**Review by Naomimmm — APPROVED** [review]
Left a couple of questions but look great!

**Review by jcasta30 — APPROVED** [review]
Other than Noemi's question it looks good!

File: mySqlDatabaseRemote/src/mySqlDriver2/SearchInventoryGUI.java (hunk 2 above)
Comment (jblss888) [reply mySqlDatabaseRemote/src/mySqlDriver2/SearchInventoryGUI.java:100]: I believe so. That way it closes but doesn't terminate the run

**Pull request merged**